lib/menu.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Menu do
+  def game_types do
+    display_game_types
+
+    get_value_betweenf(0, Enum.count(games) - 1)
```

Comment:
And please use `games()` otherwise its hard to tell that it is a function, not a variable